msmbuilder/tests/test_strucrmsdfeaturizer.py

```diff
@@ -7,18 +7,28 @@
 def test_alanine_dipeptide():
     # This test takes the rmsd of the 0th set of alanine dipeptide
     # trajectories relative to the 0th frame of the dataset.
-    # The test asserts that the first rmsd calculated will be zero.
+    # The test asserts that the first rmsd calculated will be zero
+    # for both all atom indices and a subset of atom indices.
+    # It then verifies that the RMSD's calculated from the different
+    # sets of atom indices are not the same.
 
     dataset = fetch_alanine_dipeptide()
     trajectories = dataset["trajectories"]
-    featurizer = StrucRMSDFeaturizer(trajectories[0], trajectories[0][0],
+    featurizer = StrucRMSDFeaturizer(trajectories[0][0],
                                      np.arange(trajectories[0].n_atoms))
     data = featurizer.transform(trajectories[0])
 
     assert (data[0] < 1e-3)
 
+    featurizer = StrucRMSDFeaturizer(trajectories[0][0],
+                                     range(trajectories[0].n_atoms)[0:11])
+    data_sliced = featurizer.transform(trajectories[0])
+
+    assert (data_sliced[0] < 1e-3)
+    assert (data[1] is not data_sliced[1])
```

Comment:
http://stackoverflow.com/questions/132988/is-there-a-difference-between-and-is-in-python